assembl/static2/js/app/components/administration/saveButton.jsx

```diff
@@ -313,10 +347,14 @@ const SaveButtonWithMutations = compose(
   }),
   graphql(updateLegalNoticeAndTermsMutation, {
     name: 'updateLegalNoticeAndTerms'
+  }),
+  graphql(updateVoteSessionMutation, {
+    name: 'updateVoteSession'
   })
 )(SaveButton);
 
 const mapStateToProps = ({
+  debate,
```

Comment:
if it is just for timeline, you could just pass `debate.debateData.timeline` in props